site2/website/data/connectors.js

```diff
@@ -13,15 +13,27 @@ module.exports = [
     },
     {
         name: 'cassandra',
-        longName: 'Apache Cassandra sink',
-        type: 'Sink',
+        longName: 'Apache Cassandra source and sink',
+        type: 'Source and sink',
```

Comment:
I am really curious if this field is used anywhere.

Reply:
I have checked it locally. The links all work well.

Reply:
This comment was not really related to  your changes. The _type_ field is not used and maybe it makes sense to remove it in the future

Reply:
Okay. Thanks so much for your reply.